ParisBNF/abb/BNFabb20.xml

```diff
@@ -97,25 +97,27 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
-            <div type="bibliography"/>
+            <div type="bibliography">
+             
+            </div>
             <div type="edition" xml:lang="gez">
                 <div type="textpart" n="1" subtype="folio">
                     <ab>
-                        <pb n="1r"/> ኆልቆሙ፡ ለእለ፡ ተጋብኡ። ለሕንፃ፡ ማኅፈድ፡ ፸ወ፪፡ መሳፍንት፡ እምደቂቀ፡ <persName ref="PRS0000">ሴም፡
-                        </persName> ፳ወ፭ብእሲ፡ ወእሉ፡ ኀደሩ፡ እም<placeName ref="LOC0000">አፍራጥስ፡ </placeName>ወእስከ፡
-                        <placeName ref="LOC0000">ሶርያ፡ </placeName>ወእስከ፡ ጽንፈ፡ ምሥራቅ፡ ወእምኔሆሙ፡ <placeName ref="LOC0000">ሶርያውያን፡
+                        <pb n="1r"/> ኆልቆሙ፡ ለእለ፡ ተጋብኡ። ለሕንፃ፡ ማኅፈድ፡ ፸ወ፪፡ መሳፍንት፡ እምደቂቀ፡ <persName ref="PRS8730Sem">ሴም፡
+                        </persName> ፳ወ፭ብእሲ፡ ወእሉ፡ ኀደሩ፡ እም<placeName ref="LOC3013Euphra">አፍራጥስ፡ </placeName>ወእስከ፡
+                        <placeName ref="LOC5833Syria">ሶርያ፡ </placeName>ወእስከ፡ ጽንፈ፡ ምሥራቅ፡ ወእምኔሆሙ፡ <placeName ref="ETH0000">ሶርያውያን፡
                         </placeName>ወ<placeName ref="LOC0000">አእፃዳተ፡ ዘረቢዐ፡ </placeName>ወደሴተ፡ <placeName ref="LOC0000">ወሐራምቃ፡ </placeName>
-                        <placeName ref="LOC0000">ከለዳውያን፡ </placeName>ወእሙንቱ፡ ሰብአ፡ <placeName ref="LOC0000">ባቢሎን፡
-                        </placeName>ወ<placeName ref="LOC0000">ፋርስ፡ </placeName> ወ<placeName ref="LOC0000">ኵራስን</placeName>፡
+                        <placeName ref="ETH0000">ከለዳውያን፡ </placeName>ወእሙንቱ፡ ሰብአ፡ <placeName ref="LOC1692Babylo">ባቢሎን፡
+                        </placeName>ወ<placeName ref="LOC3930Iran">ፋርስ፡ </placeName> ወ<placeName ref="LOC0000">ኵራስን</placeName>፡
                         ወ<placeName ref="LOC0000">ፌርያ</placeName>፡ ወ<placeName ref="LOC0000">ስንድ</placeName>፡
-                        ወ<placeName ref="LOC0000">ህንዳኬ</placeName>፡ ወ<placeName ref="LOC0000">የማን</placeName>፡
+                        ወ<placeName ref="LOC0000">ህንዳኬ</placeName>፡ ወ<placeName ref="LOC0301IHA">የማን</placeName>፡
```

Comment:
I believe it is best not to use IHA records (I know it is very confusing that they show up in the search). There is also LOC6387Yemen (although for the modern state)

Reply:
we may not use IHA records at all. they do not belong to BM. but of course it is our fault that we have not explained this earlier.

Reply:
If we ever get there to improving our search then I think it should be possible to by default exclude IHA and only include it if needed... But we are not there yet...